swagger.yaml

```diff
@@ -6221,6 +6221,11 @@ definitions:
         default: "x86_64"
         description: The CPU architecture of the image (x86_64/arm64/etc).
```

Comment:
Maybe add here 'deprecated' mark, just as a note to remove it later on.

Reply:
Right, added `(DEPRECATED)` as a prefix